src/psyclone/tests/nemo/nemo_psy_test.py

```diff
@@ -45,6 +45,7 @@
 from psyclone.tests.utilities import get_invoke
 from psyclone import nemo
 from fparser.common.readfortran import FortranStringReader
+from psyclone.psyir.nodes import Assignment, CodeBlock
```

Comment:
pylint complains that imports from `psyclone` are not grouped.

Reply:
Fixed.